Sources/Misc/Concurrency/OperationDispatcher.swift

```diff
@@ -57,7 +57,7 @@ class OperationDispatcher {
         Self.dispatchOnMainActor(block)
     }
 
-    func dispatchOnWorkerThread(jitterableDelay delay: JitterableDelay = .none, block: @escaping @Sendable () -> Void) {
+    func dispatchOnWorkerThread(delay: JitterableDelay = .none, block: @escaping @Sendable () -> Void) {
```

Comment:
I'm not sure what's going on with GitHub's diff rendering here, the `jitterableDelay` isn't present in the `retry-429` branch: https://github.com/RevenueCat/purchases-ios/blob/3b2018698e046295fc045c343ffb6021b88baa05/Sources/Misc/Concurrency/OperationDispatcher.swift#L60